CSS/main.css

```diff
@@ -2,13 +2,20 @@
 @import './authenticate.css';
 @import './aside.css';
 @import './wishlist.css';
+@import './cart.css';
 
 .short-width {
     width: 80%;
     margin-left: auto;
     margin-right: auto;
```

Comment:
Shorthand property could be used here which goes this way
`margin: initial auto`

Reply:
Thanks for the suggestion @VishalPatil18